experimental/seaweedfs/base/pipeline-profile-controller/sync.py

```diff
@@ -0,0 +1,445 @@
+                        "METADATA_GRPC_SERVICE_PORT": "8080",
+                    },
+                },
+                {
+                    "apiVersion": "v1",
+                    "kind": "ConfigMap",
+                    "metadata": {
+                        "name": "artifact-repositories",
+                        "namespace": namespace,
+                        "annotations": {
+                            "workflows.argoproj.io/default-artifact-repository": "default-seaweedfs-namespaced"
+                        }
+                    },
+                    "data": {
+                        "default-seaweedfs-namespaced": json.dumps({
+                            "archiveLogs": True,
+                            "s3": {
+                                "endpoint": "minio-service.kubeflow:9000",
+                                "bucket": "kubeflow",
+                                "keyFormat": f"{namespace}/artifacts/{{{{workflow.name}}}}/{{{{workflow.creationTimestamp.Y}}}}/{{{{workflow.creationTimestamp.m}}}}/{{{{workflow.creationTimestamp.d}}}}/{{{{pod.name}}}}",
+                                "insecure": True,
+                                "accessKeySecret": {
+                                    "name": "mlpipeline-minio-artifact",
+                                    "key": "accesskey",
+                                },
+                                "secretKeySecret": {
+                                    "name": "mlpipeline-minio-artifact",
+                                    "key": "secretkey",
+                                }
+                            }
+                        })
+                    }
+                },
+                # Visualization server related manifests below
+                {
+                    "apiVersion": "apps/v1",
+                    "kind": "Deployment",
+                    "metadata": {
+                        "labels": {
+                            "app": "ml-pipeline-visualizationserver"
+                        },
+                        "name": "ml-pipeline-visualizationserver",
+                        "namespace": namespace,
+                    },
+                    "spec": {
+                        "selector": {
+                            "matchLabels": {
+                                "app": "ml-pipeline-visualizationserver"
+                            },
+                        },
+                        "template": {
+                            "metadata": {
+                                "labels": {
+                                    "app": "ml-pipeline-visualizationserver"
+                                },
+                                "annotations": disable_istio_sidecar and {
+                                    "sidecar.istio.io/inject": "false"
+                                } or {},
+                            },
+                            "spec": {
+                                "containers": [{
```

Comment:
please add a securitycontext as in https://github.com/kubeflow/manifests/pull/3050 for the deprecated visualization-server and artifact-proxy